tests/ssl-opt.sh

```diff
@@ -1860,6 +1860,48 @@ run_test    "Opaque key for server authentication: invalid alg: TLS-ECDHE-ECDSA
             -s "error" \
             -c "error"
 
+requires_config_enabled MBEDTLS_SSL_PROTO_TLS1_2
+requires_config_enabled MBEDTLS_USE_PSA_CRYPTO
+requires_config_enabled MBEDTLS_X509_CRT_PARSE_C
+requires_config_enabled MBEDTLS_ECDSA_C
+requires_config_enabled MBEDTLS_SHA256_C
+run_test    "Two Opaque keys for server authentication: EC with different algs" \
+            "$P_SRV auth_mode=required key_opaque=1 crt_file=data_files/server5.crt \
```

Comment:
Minor: since we're testing server auth, we could remove `auth_mode=required` here and `crt_file`/`key_file` in the client command line in order to reduce clutter.